src/gas/GasFeeController.ts

```diff
@@ -388,66 +368,35 @@ export class GasFeeController extends BaseController<
       isEIP1559Compatible = false;
     }
 
-    let newState: GasFeeState = {
-      gasFeeEstimates: {},
-      estimatedGasFeeTimeBounds: {},
-      gasEstimateType: GAS_ESTIMATE_TYPES.NONE,
-    };
-
-    try {
-      if (isEIP1559Compatible) {
-        const estimates = await this.fetchGasEstimates(
-          this.EIP1559APIEndpoint.replace('<chain_id>', `${chainId}`),
-          this.clientId,
-        );
-        const {
-          suggestedMaxPriorityFeePerGas,
-          suggestedMaxFeePerGas,
-        } = estimates.medium;
-        const estimatedGasFeeTimeBounds = this.getTimeEstimate(
```

Comment:
Huh. In the previous version, we were getting time estimates using `this.getTimeEstimates`, which used the estimates on `this.state.gasFeeEstimates` to base them on. i.e. it was using the _old_ estimates, from the previous update. Seems like a bug! Though you've now fixed it in the new `determineGasFeeSuggestions` function.

Reply:
Ah, true! I knew that `calculateTimeEstimate` took a third argument that `getTimeEstimate` didn't take, but it didn't register that `getTimeEstimate` was relying on old data. Great :)